app/models/studio.rb

```diff
@@ -1,3 +1,12 @@
 class Studio < ApplicationRecord
   has_many :movies
+  has_many :actors, through: :movies
```

Comment:
Not many people know you can do this, and it took me a while to understand this when I first encountered it. Great job making use of Rails to get to `:actors` through the association that the `Movie` model already has through `:movie_actors`!